extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleJobStore.java

```diff
@@ -351,12 +371,19 @@ private Entity createUpdatedEntity(Key key, Map<String, Object> data)
     return createEntityBuilder(key, data).set(LAST_UPDATE_FIELD, Timestamp.now()).build();
   }
 
-  private Key getKey(UUID jobId) {
-    return datastore.newKeyFactory().setKind(KIND).newKey(jobId.toString());
+  private Key getJobKey(UUID jobId) {
+    return datastore.newKeyFactory().setKind(JOB_KIND).newKey(jobId.toString());
+  }
+
+  private Key getErrorKey(UUID jobId, String errorId) {
+    return datastore.newKeyFactory()
+        .setKind(ERROR_KIND)
+        .addAncestor(PathElement.of(JOB_KIND, jobId.toString()))
```

Comment:
I know it's trivial but let's comment or link to the Datastore doc describing this relationship

Reply:
Done.